cli/src/command/other.rs

```diff
@@ -23,13 +23,21 @@ use super::*;
 pub enum Command {
     /// Show the genesis hash the node uses
     GenesisHash(ShowGenesisHash),
+
+    /// Update the WASM runtime running on the node it
+    /// will connect to. Requires the `spec_version` of
```

Comment:
I think it is important to point out that this is a transaction that modifies the chain state.
```suggestion
    /// Submit a transaction to update the on-chain runtime.
    /// Requires the `spec_version` of
```

We should also point out that the author must match the chains sudo key.